yaramo/signal.py

```diff
@@ -24,7 +25,13 @@ class SignalFunction(Enum):
     Einfahr_Signal = 0
     Ausfahr_Signal = 1
     Block_Signal = 2
-    andere = 3
+    Vorsignal_Vorsignalwiederholer = 3
+    Zwischen_Signal = 4
+    # Not real signals: signal.kind == SignalKind.FikitivesSignal
+    Zug_Ziel_Strecke = 80
```

Comment:
Is this a *Richtungsanzeiger*? Since we use German identifiers anyway, we could consider using the established names (the following could be *Rangiersignal*?),

Reply:
Both are fictional signals, that only exists in PlanPro due to the fact, that for example routes needs a start and end signal. So I don't think that these are _Richtungsanzeiger_ or _Rangiersignal_

Reply:
I see, thx for explanation.